### DISCUSSION TITLE
Run query periodically, store results, graph evolution (like Munin)

### DISCUSSION THREAD

**Discussion by matcho** [getredash/redash#6484] [Q&A] [answered]
Hello,

Is it possible to do the following using Redash ?

 * schedule a query that returns for ex. a single value, representing an indicator at the time the query is run
 * store the result every time the query is run
 * get a chart that represents the indicator value over time (x: time the query was run, y: result at that time)

This would be useful to keep track of the evolution of certain indicators that cannot be evaluated a-posteriori for a given period of time (ie. cannot go back in time).

Thank you

**Comment by justinclift** [comment, marked as answer]
At present, Redash isn't structured in a way that would easily do this.

It's the kind of thing that would probably be do-able in the data source itself instead (aka run the query and store the data there), with Redash being used to generate the final chart showing the change over time instead.

**Comment by matcho** [comment]
That's good to know, thank you.
